README.md

```diff
@@ -77,7 +77,6 @@ Syntax:
 |[Atharva Ikhar](https://twitter.com/iatharva_)| Happy Birthday, Eddie🥳🎉! Thanks a lot for all the suggestions you have given me, and all the efforts which you put into your videos, Have a great birthday!|
 |[Krish Gupta](https://twitter.com/krshkodes)|Happy Birthday Eddie! 🎂 Wishing you a healthy life with lots of sleep, you do a lot for us, this is probably the year you should take care of your self too :) Happy birthday once again, have a great life ahead 🚀!|
 |[Anas Khan](https://twitter.com/anaskhan_28)| Happiest Birthday, Eddie!🥳🥳 I wish you a great journey in open source and always help others to get in 😎.|
```

Comment:
```suggestion
|[Anas Khan](https://twitter.com/anaskhan_28)| Happiest Birthday, Eddie!🥳🥳 I wish you a great journey in open source and always help others to get in 😎.|
|[Krishna Agarwal](https://twitter.com/DMKrishnaA)| Happy Birthday Eddie 🥳, Thanks a lot for helping us being better and I hope you get successful and Bless you always 🙏🏻.|
|[Vaibhav Malhotra](https://github.com/vaibhavmalhotra002)| Happy Birthday Eddie 🎈🎂 ,Thank you for motivating us to start contributiong to open source|
|[Raj Shukla](https://twitter.com/rajshuk49989182)| Happy Birthday, Eddie! Wish you a very very happy birthday brother may you get all you want and thanks for being so open to help all of us newbies of open source|
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix